apps/client/src/features/viewers/backstage/Backstage.tsx

```diff
@@ -146,30 +141,23 @@ export default function Backstage(props) {
         </AnimatePresence>
       </div>
 
-      <ScheduleProvider
-        events={filteredEvents}
-        selectedEventId={selectedId}
-        isBackstage
-      >
+      <ScheduleProvider events={filteredEvents} selectedEventId={selectedId} isBackstage>
         <ScheduleNav className='schedule-nav-container' />
         <Schedule className='schedule-container' />
       </ScheduleProvider>
 
-      <div
-        className={showPublicMessage ? 'public-container' : 'public-container public-container--hidden'}>
+      <div className={showPublicMessage ? 'public-container' : 'public-container public-container--hidden'}>
         <div className='label'>Public message</div>
         <div className='message'>{publ.text}</div>
       </div>
 
       <div className='info'>
         <div className='qr'>
-          {general.url != null && general.url !== '' && (
-            <QRCode value={general.url} size={qrSize} level='L' />
+          {general.backstageUrl != null && general.backstageUrl !== '' && (
```

Comment:
now that we have typescript we can be more confident about the data type, so

`{general.backstage && (`